### PR TITLE
Several minor fixes and improvements

### PR DESCRIPTION
**Pull request by march08**

### CODE REVIEW

File: src/pages/Home/CoinEarnings.tsx [hunk 1]
```diff
@@ -268,14 +277,18 @@ export const CoinEarnings = () => {
...
-              <Button variant="primary" as={LinkOut} href={TELEGRAM_LINK}>
-                <FaTelegram /> &nbsp; Telegram
-              </Button>{' '}
-              &nbsp;
-              <Button variant="warning" as={LinkOut} href={DISCORD_LINK}>
-                <FaDiscord />
-                &nbsp;Discord
-              </Button>
+              <ButtonGroup>
+                <Button variant="danger" as={LinkOut} href={TELEGRAM_LINK}>
```
Comment (AlexSSD7) [inline src/pages/Home/CoinEarnings.tsx:281]: `REDDIT_LINK` should be here.

**Review by AlexSSD7 — CHANGES_REQUESTED** [review]
Reddit link is pointing to Telegram. All other looks ok.

File: src/pages/Home/CoinEarnings.tsx (hunk 1 above)
Comment (march08) [reply src/pages/Home/CoinEarnings.tsx:281]: oh right thanks

**Review by AlexSSD7 — APPROVED** [review]
LGTM

**Pull request merged**